### ISSUE TITLE
Suggestion: Disable Tx with keyboard lock

### ISSUE THREAD

**Issue by SKUSKAskuska** [egzumer/uv-k5-firmware-custom#94]
Hello - first to begin with - thanks for the great work! I would have one suggestion, I don't know if it is feasible.

It would be great if locking the keyboard (long press F) would also disable Tx via PTT.

Sometimes I carry the radio hanging on beltclip or in backpack only for Rx, and it is necessary to be very careful not to press the PTT while walking, because by default it is active even when the keyboard is locked.
Maybe some people prefer transmitting with keyboard locked, alternatively, on several of TYT handhelds it is solved by menu item for LOCK regime (key, PTT, key+PTT).

**Comment by ErikS-web** [comment]
For now, you can use 61 `F Lock` - sets the TX frequency band plan (see [Hidden menu](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Menu#hidden-menu))
and set it to "DISABLE ALL - disables TX on all frequencies"

**Comment by karacsonyisandor** [comment]
Yes, but you have to enter the hidden menu every time to enable/disable... not practical.

**Comment by ErikS-web** [comment]
As long as the radio stays ON, the hidden menu items can be accessed

**Comment by karacsonyisandor** [comment]
@SKUSKAskuska  https://github.com/karacsonyisandor/uvk5-mods/commits/main/firmware.packed.bin 
moded the firmware so the hidden menu is always on. You can disable the tx with F Lock quickly.

**Comment by SKUSKAskuska** [comment]
Great, thanks!

**Comment by RoadRangerX** [comment]
Just wanted to say that I also would use this enhancement if it is ever implemented :-)

**Comment by var-tz** [comment]
uvk5's ptt button is very easy to mistouch. sometimes i want lock it , sometimes i want speak someting,  use (lock option with tx) as a quick mode is a good idea.  use hidden menu is a bad idea.

**Comment by gg33fr** [comment]
Hello, I don't understand the procedure to unblock in tx
I'm in the hidden menu, I go to F-lock, unlock all, I press M to validate and I have read manual, I come out and I go back to F-lock and unlock all or I do the procedure 10 times but it doesn't work, I always have tx-disable
Can someone give me the exact procedure with the keys to press in the exact order?
THANKS
Happy New Year

**Comment by KvallVall** [comment]
A dumb workaround I use is to set the cursor on something in AM mode (can't transmit on AM) and lock the keypad.

**Comment by BruMcD** [comment]
Mostly the keyboard lock is used to put down /pass the set out for use without you/users knocking themselves off channel etc in a car, bag or pocket and losing contact  (and it persists after switch off  /on for obvious reasons) 

The PTT / monitor /volume etc.   all still working on the right channel if needed is the whole point of it.

If locking everything is needed it should be another sort of lock to add to  the current on/off Lock
